packages/TelegramClient-UI.package/TCUMessage.class/class/newFrom..st

```diff
@@ -0,0 +1,10 @@
+instance creation
+newFrom: aCoreMessage
+
+	^ (aCoreMessage class caseOf: {
+		[TCCTextMessage] -> [TCUTextMessage].
+		[TCCPhotoMessage] -> [TCUPhotoMessage].
+	} otherwise: [TCUTextMessage])
+	basicNew
+	messageModel: aCoreMessage;
+	initialize
```

Comment:
please indent cascade and add `yourself`

Reply:
fixed